test/odh/notebook.go

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 package odh
 
+/*
```

Comment:
Just wondering why this needs to be commented out?

Reply:
they aren't used because I removed the MCAD-based test that used them.  So the linter failed.  I can remove the helper file too if you'd prefer.

Reply:
I think there are run for downstream testing on OpenShift. So the less that's commented out the better. They'll have to be updated to use the SDK without MCAD.